### ISSUE TITLE
删除日志报错

### ISSUE THREAD

**Issue by JDongKhan** [HDB-Li/LLDebugTool#21]
DELETE FROM LLLogModelTable_1 WHERE Identity IN 22 (
2018-08-17 11:46:1956,
2018-08-17 11:47:1275,
2018-08-17 11:46:062,
2018-08-17 11:46:063,
2018-08-17 11:46:0914,
2018-08-17 11:46:064,
2018-08-17 11:46:4067,
2018-08-17 11:46:0916,
2018-08-17 11:46:067,
2018-08-17 11:46:066,
2018-08-17 11:46:068,
2018-08-17 11:46:0917,
2018-08-17 11:46:069,
2018-08-17 11:46:0918,
2018-08-17 11:46:051,
2018-08-17 11:46:1124,
2018-08-17 11:46:4669,
2018-08-17 11:46:1353,
2018-08-17 11:46:0919,
2018-08-17 11:46:0920,
2018-08-17 11:46:1126,
2018-08-17 11:46:2861,
);

sql如上

**Comment by JDongKhan** [comment]
哦，是不是需要APP卸载重装！

**Comment by JDongKhan** [comment]
应该是多了个22，数组使用%@格式化会带有数量

**Comment by HDB-Li** [comment]
Did use some libs which recode [NSArray description] ?    I just try in Xcode 9.3 ,iOS 11.4, iPhone 8 Plus, 
my SQL is 
```
DELETE FROM LLCrashModelTable WHERE Identity IN (
    "2018-08-17 12:00:13",
    "2018-08-17 12:00:31"
);

```   

there is no count between `IN` and `(`.

**Comment by JDongKhan** [comment]
是的，其他库导致的

**Comment by JDongKhan** [comment]
之前有说过在低版本比如iPhone5 8.1下，NSURLSessionConfiguration的protocolClasses设置导致网络失败，现在发现是返回的json格式出问题了，数据出现两份不是一个完整的json格式。具体原因还在看。

**Comment by HDB-Li** [comment]
@zou145688zhuang  OK I will fix it , Thanks. 

One more question, How did you make `[UIImage imageWithData:obj]` return a NSNull object?

```
- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
    NSString *title = self.titleArray[indexPath.row];
    if ([self.canCopyArray containsObject:title]) {
        id obj = self.contentArray[indexPath.row];
        if ([obj isKindOfClass:[NSData class]] && self.model.isImage) {
            UIImage *image = [UIImage imageWithData:obj];
            [UIPasteboard generalPasteboard].image = image;
        } else {
            if ([obj isKindOfClass:[NSData class]]) {
                obj = [self convertDataToHexStr:obj];
            }
            [UIPasteboard generalPasteboard].string = obj;
        }
        [self toastMessage:[NSString stringWithFormat:@"Copy \"%@\" Success",title]];
    }

}
```

**Comment by HDB-Li** [comment]
@JDongKhan  hi JDong,  Sorry , I just saw what you said.
```
之前有说过在低版本比如iPhone5 8.1下，NSURLSessionConfiguration的protocolClasses设置导致网络失败，现在发现是返回的json格式出问题了，数据出现两份不是一个完整的json格式。具体原因还在看...
```
Have you found a problem with other libraries? (because I use the runtime in `NSURLSessionConfiguration`) or error happened when I append data?

```
还有一种情况是数据很大的时候会截掉！
```
I recently solved a problem that when the data is too large, such as 1000 rows, it will lead to incomplete display in LLDebugTool, but it will not affect the reception of the data. I don't know if you are talking about this problem.
